mattermost-plugin/webapp/package.json

```diff
@@ -76,6 +76,7 @@
     "core-js": "3.12.1",
     "mattermost-redux": "5.33.1",
     "react-intl": "^5.13.5",
+    "react-markdown": "7.0.0",
```

Comment:
Is there a plan to also use `react-markdown` for card content in the main app (not plugin) instead of using `marked.js` and `dangerouslySetInnerHTML `.

Reply:
To be honest, probably not. I was just googling a markdown library i can use for the sake of displaying content in the card. But when reviews come, I'll probably be told to switch it or something. Didn't look at what we are using and just decided first result in google haha

Reply:
FYI, both of those differ from the web app and mobile app Markdown since they used forked versions of marked.js and commonmark.js respectively. I'm still hoping to get those all sorted out and available for use in plugins eventually though.